inc/customizer/settings.php

```diff
@@ -53,6 +53,34 @@ function _customize_additional_scripts( $wp_customize ) {
 add_action( 'customize_register', '_customize_additional_scripts' );
 
 /**
+ * Register settings for site identity
+ * The section title_tagline is provided by Largo under the label "Site Identity"
+ */
+function _customize_title_tagline( $wp_customize ) {
+	// Register a setting
+	$wp_customize->add_setting(
+		'site_blurb',
+		array(
+			'default'           => '',
+			'sanitize_callback' => 'wp_filter_nohtml_kses',
```

Comment:
I like [wp_kses_data](https://developer.wordpress.org/reference/functions/wp_kses_allowed_html/) here. That will limit it to the tags contained in the global `$allowedtags` array (https://core.trac.wordpress.org/browser/tags/4.7.3/src/wp-includes/kses.php#L424), which has common text formatting but not images, lists, videos, or block quotes.

Or we could just use `wp_kses_post`, which would allow everything used on posts.